Resources/Prototypes/Entities/Objects/Specific/Hydroponics/tools.yml

```
@@ -107,3 +107,28 @@
         Piercing: 5 # I guess you can stab it into them?
   - type: Item
     sprite: Objects/Tools/Hydroponics/spade.rsi
+
+- type: entity
+  name: plant bag
+  id: PlantBag
+  parent: BaseItem
+  description: A bag for botanists to easily move their huge harvests.
+  components:
+  - type: Sprite
+    netsync: false
+    sprite: Objects/Specific/Hydroponics/Equipment/plant_bag.rsi
+    state: icon
+  - type: Clothing
+    sprite: Objects/Specific/Hydroponics/Equipment/plant_bag.rsi
+    QuickEquip: false
+    Slots:
+    - belt
+    size: 200
+  - type: Storage
+    capacity: 200
+    quickInsert: true
+    areaInsert: true
+    whitelist:
+      tags:
+        - Produce
+        - Seed
```

you can probably whitelist by `ProduceComponent` and `SeedComponent` instead and remove the tags you added

```suggestion
    whitelist:
      components:
        - Produce
        - Seed
```